cartridge/auth-backend.lua

```diff
@@ -97,6 +138,15 @@ end
 local function add_user(username, password, fullname, email)
     checks('string', 'string', '?string', '?string')
 
+    if username == ADMIN_USER.username
+    and is_builtin_admin_enabled()
```

Comment:
Why do you discard `get_admin_username` function?

Reply:
I think `get_admin_username` name is misleading. It implies one can `set_admin_username`, but that's not true. In fact, `get_admin_username` is a twisted `is_enabled`. It's the Occam's razor: entities should not be multiplied beyond necessity.

Reply:
I don't insist on this particular name, it can be renamed. 

> entities should not be multiplied beyond necessity.
As a result, there are code duplications

Reply:
How following case will be handled?
  - Disable admin user
  - Add custom admin
  - Enable admin user